packages/xo-vmdk-to-vhd/src/vmdk-generate.js

```diff
@@ -81,8 +83,8 @@ ddb.geometry.cylinders = "${geometry.cylinders}"
 
   let streamPosition = 0
   let directoryOffset = 0
-
-  const roundToSector = value => Math.ceil(value / SECTOR_SIZE) * SECTOR_SIZE
+  const endMetadataLength = computeEndMetadataLength()
```

Comment:
This var name is unclear to me, *end* makes me thing of an offset and *length* of a size in byte.

Am I mistaken or can this name be made clearer?

Reply:
that is the length in bytes of the metadata at the enf of the file, after the blocks. I am open to suggestion

Reply:
Maybe *footer*? :thinking:

Reply:
Bad idea then :sweat_smile: 

`eofMetadataSize`?